### PR TITLE
Feature: Approve/Reject Design

### LINKED ISSUE #467: [MicroCredit] add approve/reject on borrowers list
dependent on https://github.com/impactMarket/backend/issues/598

### PR DESCRIPTION
**Pull request by marcusats**
This PR fixes #467 

## Changes

- 

## Tests

- [ ] Tested on mobile
- [ ] Tested on dektop

This PR should be manually tested by @obernardovieira @hugomarquesdev @PauloSousapt , included on reviewers.

### CODE REVIEW

**Comment by hugomarquesdev** [comment]
@marcusats eslint new error:
```
yarn run v1.22.19
$ /home/circleci/project/node_modules/.bin/tsc
src/views/MicrocreditManager/RequestTab/RequestList.tsx:360:22 - error TS6133: 'data' is declared but its value is never read.

360             render: (data : any) => (
                         ~~~~


Found 1 error in src/views/MicrocreditManager/RequestTab/RequestList.tsx:360

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Exited with code exit status 2
CircleCI received exit code 2
```

**Comment by hugomarquesdev** [comment]
@marcusats new lint errors:
```
#!/bin/bash -eo pipefail
yarn eslint:dryrun
yarn run v1.22.19
$ eslint "./**/*.{ts,tsx}" --max-warnings=0

/home/circleci/project/src/views/MicrocreditManager/RequestTab/RequestList.tsx
  34:1   error    Imports should be sorted alphabetically  sort-imports-es6-autofix/sort-imports-es6
  34:10  warning  'da' is defined but never used           @typescript-eslint/no-unused-vars

✖ 2 problems (1 error, 1 warning)
  1 error and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Exited with code exit status 1
CircleCI received exit code 1
```

Have you tried to run `yarn lint` or `yarn eslint:dryrun`? these are all lint errors.
Anyway vscode should warn you automatically when there are these errors 🤔

File: src/modals/Microcredit/AddNote.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import useTranslations from '../../libs/Prismic/hooks/useTranslations';
+
+const AddNote = () => {
+    const { extractFromView } = usePrismicData();
+    const {describeConvBorrower, addNote} = extractFromView('messages') as any;
+
+    const { handleClose } = useModal();
+    const { t } = useTranslations();
+
+    console.log("Describe",describeConvBorrower);
```
Comment (hugomarquesdev) [inline src/modals/Microcredit/AddNote.tsx:20]: 👀

File: src/modals/Microcredit/AddNote.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    >
+                        {t('save')}
+                    </Button>
+                </Box>
+            </Box>
+        </ModalWrapper>
+    );
+};
+
+export default AddNote;
```
Comment (hugomarquesdev) [inline src/modals/Microcredit/AddNote.tsx:82]: missing an empty line on the end of the file

File: src/hooks/useMicrocreditApplications.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    const loadingApplications = !data && !error;
+
+    return {
+        applications: data?.data?.rows,
+        count: data?.data?.count,
+        loadingApplications,
+        mutate
+    };
+}
```
Comment (hugomarquesdev) [inline src/hooks/useMicrocreditApplications.tsx:37]: missing an empty line on the end of the file

File: src/components/FlexibleTab.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                        number={tab.number > 0 ? tab.number : null}
+                        onClick={tab.onClick}
+                    />
+                ))}
+            </TabList>
+        </Tabs>
+    );
+};
+
+export { FlexibleTab };
```
Comment (hugomarquesdev) [inline src/components/FlexibleTab.tsx:31]: missing an empty line on the end of the file

File: src/modals/Microcredit/ApproveLoan.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    >
+                        {t('save')}
+                    </Button>
+                </Box>
+            </Box>
+        </ModalWrapper>
+    );
+};
+
+export default ApproveLoan;
```
Comment (hugomarquesdev) [inline src/modals/Microcredit/ApproveLoan.tsx:81]: missing an empty line on the end of the file

File: src/views/MicrocreditManager/BorrowersTab/BorrowersList.tsx [hunk 6]
```diff
@@ -327,4 +330,4 @@ const BorrowersList: React.FC<{
     );
 };
 
-export default BorrowersList;
+export default BorrowersList;
```
Comment (hugomarquesdev) [inline src/views/MicrocreditManager/BorrowersTab/BorrowersList.tsx:333]: missing an empty line on the end of the file

File: src/views/MicrocreditManager/BorrowersTab/index.tsx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+                    setItemOffset={setItemOffset}
+                />
+            </Box>
+        </Box>
+
+    )
+
+}
+
+export default BorrowersTab;
```
Comment (hugomarquesdev) [inline src/views/MicrocreditManager/BorrowersTab/index.tsx:100]: missing an empty line on the end of the file

File: src/views/MicrocreditManager/RequestTab/RequestList.tsx [hunk 8]
```diff
@@ -0,0 +1,442 @@
...
+            page={page}
+            count={count}
+            pb={6}
+            prefix={applications}
+            setItemOffset={setItemOffset}
+        />
+    );
+};
+
+export default RequestList;
```
Comment (hugomarquesdev) [inline src/views/MicrocreditManager/RequestTab/RequestList.tsx:442]: missing an empty line on the end of the file

File: src/views/MicrocreditManager/RequestTab/index.tsx [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+                    loadingApplications={loadingApplications}
+                    applications={applications}
+                    setItemOffset={setItemOffset}
+                />
+            </Box>
+        </Box>
+    );
+};
+
+export default RequestTab;
```
Comment (hugomarquesdev) [inline src/views/MicrocreditManager/RequestTab/index.tsx:161]: missing an empty line on the end of the file

**Review by hugomarquesdev — CHANGES_REQUESTED** [review]

**Review by obernardovieira — APPROVED** [review]

**Review by hugomarquesdev — APPROVED** [review]

**Pull request merged**